css/style.css

```diff
@@ -25,6 +25,20 @@
 	padding: 10px;
 	margin-top: 0px;
 }
+.dragDropDisplay {
+  background: linear-gradient(to left, rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.2)), url('/images/grass_block.png') top center repeat-x;
```

Comment:
Is that image a copy of the Minecraft grass block texture or some other potentially copyrighted assets of the game? 🚨

Reply:
This image is only a placeholder image, anyways I am going to change it with a better image, and will keep the copyrighted content in mind